crates/biome_js_analyze/tests/specs/correctness/noUndeclaredDependencies/invalid.js.snap

```diff
@@ -56,6 +60,62 @@ invalid.js:3:1 lint/correctness/noUndeclaredDependencies ━━━━━━━
   > 3 │ require("notInstalled")
       │ ^^^^^^^^^^^^^^^^^^^^^^^
     4 │ 
+    5 │ import "@testing-library/react";
+  
+  i This could lead to errors.
+  
+  i Add the dependency in your manifest.
+  
+
+```
+
+```
+invalid.js:5:8 lint/correctness/noUndeclaredDependencies ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! The current dependency isn't specified in your package.json.
```

Comment:
I think we need an extension for the diagnostics, because this may be triggered on dependencies that *are* in your `package.json`, just not in the right section. I’d suggest something like: “X is part of your `devDependencies`, but `devDependencies` are not available to this file. You may want to consider moving X to the `dependencies` section.”

Reply:
Nice!